app/templates/macros/page_macros.html

```diff
@@ -70,9 +78,10 @@
             cancelEdit();
         });
 
-        /*$("#" + editorIDName).focusout(function() {
-            cancelEdit();
-        });*/
+        //Manages the focus activity of the CKEditor
```

Comment:
I think we want to comment something about how we want the blue focus border to appear even if we focus out to denote an edit state? since this can be confusing to others without context